cli/.editorconfig

```diff
@@ -0,0 +1,11 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
```

Comment:
Understand. Then I will leave this `.editorconfig` as is and update the root one in order to use the 2 spaces too (all of this in another PR). In this particular project both are web related but at least opens the door to do something else.